stdlib/package.py

```diff
@@ -293,15 +293,16 @@ def move(self, srcs: str, dst: str):
         :param dst: A path pointing to the destination folder, relative to the ``wrap_cache`` of this :py:class:`.Package`.
         """
 
-        if os.path.isabs(srcs) or os.path.isabs(dst):
+        if os.path.isabs(dst):
             raise RuntimeError("Package.move() received an absolute path as parameter, but it expects a relative one")
```

Comment:
I know this is not introduced by the current commit, but shouldn't you raise `ValueError` instead ?

Reply:
I probably should, haven't spent much time looking at all possible errors in Python